staging/src/k8s.io/api/bulk/v1alpha1/types.go

```diff
@@ -0,0 +1,216 @@
+/*
```

Comment:
@smarterclayton @liggitt - can you please take a look into types definition.
All changes here may affect implementation, so I would like to ensure that we're on the same page about it.